e2e/test/api.spec.ts

```diff
@@ -607,7 +607,7 @@ describe('browser.electron', () => {
         const mockGetName = await browser.electron.mock('app', 'getName');
         const mockImpl = mockGetName.getMockImplementation() as () => undefined;
 
-        expect(mockImpl()).toBeUndefined();
+        expect(mockImpl).toBeUndefined();
```

Comment:
Hahaha was that it.  I didn't get round to looking at the changelog yet.  Thanks :)